configure.ac

```diff
@@ -487,6 +487,63 @@ if test -n "$with_os_release" && test ! -f "$with_os_release"; then
 fi
 AC_DEFINE_UNQUOTED([OSRELEASEFILE], ["$with_os_release"], [File with OS release details.])
 
+
+default_trace_program=no
+case $my_htop_platform in #(
+   linux)
+      default_trace_program=strace
+      ;;
+esac
+AC_ARG_WITH([trace_program],
+            [AS_HELP_STRING([--with-trace-program=no|strace],
+                            [program used for syscall tracing])],
+            [],
+            [with_trace_program=$default_trace_program])
+if test "$with_trace_program" != no; then
+   case $with_trace_program in #(
+      */*)
+         AC_DEFINE([TRACE_PROGRAM_NAME_WITH_SLASH], [1], [trace program name contains slashes])
+         ;;
+   esac
+   trace_program_shortname=$(printf '%s' "$with_trace_program"| sed 's|.*/||')
+
+   case $trace_program_shortname in #(
+      *strace)
+         : "${TRACE_PROGRAM_ARGS=-T -tt -s 512}"
+         ;; #(
+      *)
+         if test "x${TRACE_PROGRAM_ARGS:-}" = x; then
+            AC_MSG_WARN([--with-trace-program '$with_trace_program' is unrecognized; you may need to specify TRACE_PROGRAM_ARGS manually])
+         fi
+         ;;
+   esac
+
+   trace_program_args_c=''
+   for arg in $TRACE_PROGRAM_ARGS; do
+      arg=$(printf '%s' "$arg"|sed 's/\\/\\\\/g; s/"/\\"/g')
```

Comment:
Isn't there native quoting support to do this?

Reply:
What "native quoting support" do you mean?